simpletuner/helpers/utils/offloading.py

```diff
@@ -68,6 +68,14 @@ def enable_group_offload_on_components(
         excluded_names.add("vae")
     if "vqvae" not in excluded_names:
         excluded_names.add("vqvae")
+    if "text_encoder" not in excluded_names:
+        excluded_names.add("text_encoder")
+    if "text_encoder_2" not in excluded_names:
+        excluded_names.add("text_encoder_2")
+    if "text_encoder_3" not in excluded_names:
+        excluded_names.add("text_encoder_3")
+    if "text_encoder_4" not in excluded_names:
+        excluded_names.add("text_encoder_4")
```

Comment:
```suggestion
    for te_idx in range(2, 5):
        te_str = f"text_encoder_{te_idx}"
        if te_str not in excluded_names:
            excluded_names.add(te_str)
```

Reply:
Also, `set.add` automatically de-duplicates, you don't really need `if ... in excluded_names:` at all.